<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timestampExtractHelpDlg</class>
 <widget class="QDialog" name="timestampExtractHelpDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>To correcly label each frame with the appropriate timestamp, the time has to be known.  It can either be extracted from the image file name, or retrieved from teh image metadate using the EXIV library, which is standard for most image file types such as JPEG. If you want to extract the timestamp from teh name, this dialog can help figure out the format.  It uses Qt standard for designating days, months and times.  For example if the timestamp is 04.05.2020, the timestamp format would be &quot; dd.MM.yyyy&quot;, and if the time was 13:47:10.234, the time format would be &quot;hh:mm:ss.zzz&quot;.  To test timestamp extraction, paste an example of your image file name (with or without extension) and fill in the format string as well as the staring index  if the time is correctly extracted you will get positive feedback.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date format string</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="timestampFormat"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>date index start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="timestampStartIndex"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filenameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="extractBtn">
     <property name="text">
      <string>Extract</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
